INSTALL-from-source.md

```diff
@@ -1,27 +1,32 @@
 # Prerequisites:
 
-Build from sources you need:
+To build from sources you need:
 
 * a previous version of remake or GNU make
-* gcc
-* gettext
-* GNU Readline
-* guile version 2.0
-* texinfo
+* A C compiler like [`gcc`](https://gcc.gnu.org/) or [`clang`](https://clang.llvm.org/)
+* [`gettext`](https://www.gnu.org/software/gettext/)
+* [GNU Readline](https://tiswww.case.edu/php/chet/readline/rltop.html) which the debugger currently requires
 
+Optionally you may want:
 
-Additionally if installing from git you need:
+* [Guile](https://www.gnu.org/software/guile/) version 2.0 or greater
 
-* git (duh)
-* autoconf
-* automake
-* autopoint
-* gzip and lzip (to compress the tarball)
+
+Additionally, if installing from git you need:
+
+* `git` (duh)
+* `autoconf`
+* `automake`
+* `autopoint`
+
+and optionally:
+
+* `gzip` and `lzip` if building a compressed tarball
 
 Here is a `apt-get` command you can use to install on Debian-ish systems:
 
 ```console
-   $ sudo apt-get install git gcc pkg-config autoconf automake autopoint libreadline-dev make guile-2.0 texinfo lzip
+   $ sudo apt-get install git gcc pkg-config autoconf automake autopoint libreadline-dev make guile-2.2 texinfo lzip
```

Comment:
On a new Debian/Ubuntu I did need the gettext package, otherwise the `msgmerge` utility couldn't be found.

Reply:
Thanks - that should be added in fcab2fc now.